schemaspy/pgsql-oracle-fdw.properties

```diff
@@ -0,0 +1,24 @@
+# Current State
+# The following configuration allows SchemaSpy to query and graph the Foreign tables imported by oracle-fdw.
```

Comment:
We don't need this.. and also I see some files in /schemaspy/conf that are duplicates of these files. Please review.

Reply:
yep, I'll look into the dupes